pandas/tests/groupby/test_apply.py

```diff
@@ -148,7 +148,7 @@ def f_scalar(group):
         names.append(group.name)
         return 0
 
-    def f_none(group):
+    def f_none(group):  # pylint: disable= "useless-return"
         # GH10519, GH12155, GH21417
         names.append(group.name)
         return None
```

Comment:
can't this one be removed as well?

Reply:
Functions above and below this function have the same functionality and differ only by return type. To make the code more readable I left `return None`.
 
Should I remove it too?

Reply:
yeah I think that's better

Reply:
Thanks. I removed it too and added fixup commit to the PR.